Signal/src/ViewControllers/Utils/NeverClearView.swift

```diff
@@ -0,0 +1,12 @@
+// Created by Fredrik Lillejordet on 04.03.2018.
+// Copyright © 2018 Open Whisper Systems. All rights reserved.
+
+@objc class NeverClearView: UIView {
+    override var backgroundColor: UIColor? {
+        didSet {
+            if backgroundColor != nil && backgroundColor!.cgColor.alpha == 0 {
```

Comment:
nit: prefer implicit unwrap instead of `!` whenever feasible.

i.e.

```
if let backgroundColor = self.backgroundColor, backgroundColor.cgColor.alpha == 0 {
...
```